networking_nsxv3/db/db.py

```diff
@@ -25,7 +25,7 @@ def _validate_one(result, error):
     if result:
         return result
     else:
-        raise exceptions.ObjectNotFound(msg.format(error))
+        raise exceptions.ObjectNotFound(id=msg.format(error))
```

Comment:
I think you want to just pass `error` here instead of `msg.format(error)`. Currently it looks like `Object 'FirewallSection=7a7sfasdf not found in Neutron' not found.`.